espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/Element.kt

```diff
@@ -137,3 +143,39 @@ class Element(view: View) {
         }
     }
 }
+
+class ComposeElement(node: SemanticsNodeInteraction) : BaseElement {
+    @Suppress("JoinDeclarationAndAssignment")
+    @SerializedName(JSONWP_ELEMENT_KEY, alternate = [W3C_ELEMENT_KEY])
+    override val element: String
+
+    init {
+        element = UUID.randomUUID().toString()
+        ComposeViewCache.put(element, node)
+    }
+
+    companion object {
+        /**
+         * Retrieve cached view and return the SemanticsNodeInteraction
+         */
+        @Throws(AppiumException::class)
+        fun getNodeInteractionById(elementId: String?): SemanticsNodeInteraction? {
```

Comment:
The result of this method should not be nullable

Reply:
fixed